linter_config.json

```diff
@@ -23,8 +23,8 @@
     "redundant return statement",
     "comment or be unexported",
     "comment on exported",
-    "pkg/apis/tensorflow/v1alpha1/zz_generated.deepcopy.go",
-    "pkg/apis/tensorflow/v1alpha1/zz_generated.defaults.go"
+    "pkg/apis/tensorflow/v1alpha2/zz_generated.deepcopy.go",
+    "pkg/apis/tensorflow/v1alpha2/zz_generated.defaults.go"
```

Comment:
How about keep both v1alpha1 and v1alpha2?

Reply:
SGTM